lib/page_model.dart

```diff
@@ -144,54 +144,50 @@ class SinglePageModel extends PageModel {
 
     if (docMap.containsKey("View")) {
```

Comment:
This block is useless and confusing. Why set rootWidgetModel at all? Why nested the if, only to execute the logic outside of it. Please refactor